docs/zkapps/getting-started-zkapps.mdx

```diff
@@ -21,16 +21,17 @@ zkApp programmability is not yet available on the Mina Mainnet. You can get star
 
 # zkApps Getting Started
 
-You can start writing zkApps with just a few steps. You can write zkApps for your project in other ways, this Getting Started Guide provides a high-level workflow to build and deploy quickly.
+You can start writing zkApps with just a few steps. You can write zkApps for your project in other ways. The focus of this Getting Started Guide is a high-level workflow to build and deploy quickly.
```

Comment:
Do we need "You can write zkApps for your project in other ways"? It reads a little strangely to me and I wonder what we gain by saying this.

Reply:
I am digging this detailed review, seeing how fresh eyes evaluate the content, thank you!